ip_reader.h

```diff
@@ -0,0 +1,62 @@
+#include <stdint.h>
+
```

Comment:
One common comment for all functions here. If you don't need to expose function in header file, leave it only in c file and make it static. If your intention is to expose functions you need to be a little bit more careful with naming. Please remember we don't have namespaces in C. Each of exposed functions should have "ipv4" in name, or common prefix.

Reply:
Made some changes to that.